packages/network/wireguard/system.d/wireguard-config.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=WireGuard Config Service
+After=local-fs.target
+ConditionPathExists=/storage/.cache/wireguard/privatekey
+ConditionPathExists=/storage/.cache/wireguard/publickey
+ConditionPathExists=/storage/.cache/wireguard/preshared
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c '/usr/bin/wg-config'
```

Comment:
If `wg-config` is executable, `/bin/sh -c` is not required here and in `wireguard.service`.

Reply:
changed